src/site/_includes/components/cookie/index.ts

```diff
@@ -0,0 +1,74 @@
+import { getCookie, setCookie } from './cookie'
+
+const COOKIE_BANNER_CLASS_PREFIX = 'js-cookie-guidance-banner'
+const COOKIE_PREFERENCE_KEY = 'cookie_preference'
+
+const getClassName = (className: string) => `.${COOKIE_BANNER_CLASS_PREFIX}--${className}`
+
+type CookieMessageToggles = {
```

Comment:
Would be good to keep some consistency about `interface` vs `type`

I've been following the rule:

> If you would like a heuristic, use interface until you need to use features from type.

from:
https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces